src/ducks/registreringstatus.ts

```diff
@@ -16,6 +16,28 @@ export enum RegistreringType {
     ORDINAER_REGISTRERING = 'ORDINAER_REGISTRERING'
 }
 
+export enum Servicegruppe {
```

Comment:
Skal vi ta med `BKART` og `OPPFI` for sikkerhetsskyld?

Reply:
legger de inn in neste runde